app/routes/auth.py

```
@@ -116,107 +61,25 @@ def register(req: RegisterRequest, db: Session = Depends(get_db)):
+        db.add(wallet)
     db.commit()
     return {"message": "Registration successful", "user_id": user.id}
 
 @router.post("/login", response_model=Token)
-def login(
-    request: Request,
-    form: OAuth2PasswordRequestForm = Depends(),
-    otp_code: str | None = Form(default=None),
-    db: Session = Depends(get_db)
-):
+def login(form: OAuth2PasswordRequestForm = Depends(), db: Session = Depends(get_db)):
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)